src/requests/all/mod.rs

```diff
@@ -124,9 +125,41 @@ pub use set_chat_title::*;
 pub use set_game_score::*;
 pub use set_sticker_position_in_set::*;
 pub use set_webhook::*;
+use std::ops::Deref;
 pub use std::pin::Pin;
 pub use stop_message_live_location::*;
 pub use stop_poll::*;
 pub use unban_chat_member::*;
 pub use unpin_chat_message::*;
 pub use upload_sticker_file::*;
+
+/// A wrapper that implements `Clone`, Copy, `PartialEq`, `Eq`, `Debug`, but
+/// performs no copying, cloning and comparison.
+///
+/// Used in the requests bodies.
+#[derive(Debug)]
+struct BotWrapper<'a>(&'a Bot);
+
+impl PartialEq for BotWrapper<'_> {
+    fn eq(&self, _: &BotWrapper<'_>) -> bool {
+        true
```

Comment:
May be worth comparing tokens?

Reply:
But what if `reqwest::Client` are different? They does not implement `PartialEq`...
I don't know should we implement `PartialEq` + `Clone` for the requests, because it comes with a bunch of problems, since all the requests contain `Bot`.

Reply:
Not need to compare `reqwest::Client`s because they are will send requests to Telegram from equal tokens, just different ways.

Reply:
Agree with you.